pkg/api/apisix/types/types.go

```diff
@@ -33,27 +33,34 @@ type Vars [][]StringOrSlice
 
 // Route apisix route object
 type Route struct {
-	ID string `json:"id" yaml:"id"`
-
+	ID          string `json:"id" yaml:"id"`
 	Name        string `json:"name" yaml:"name"`
 	Labels      Labels `json:"labels,omitempty" yaml:"labels,omitempty"`
 	Description string `json:"desc,omitempty" yaml:"desc,omitempty"`
 
 	Host            string           `json:"host,omitempty" yaml:"host,omitempty"`
 	Hosts           []string         `json:"hosts,omitempty" yaml:"hosts,omitempty"`
 	Uri             string           `json:"uri,omitempty" yaml:"uri,omitempty"`
+	Uris            []string         `json:"uris,omitempty" yaml:"uris,omitempty"`
 	Priority        int              `json:"priority,omitempty" yaml:"priority,omitempty"`
 	Timeout         *UpstreamTimeout `json:"timeout,omitempty" yaml:"timeout,omitempty"`
 	Vars            Vars             `json:"vars,omitempty" yaml:"vars,omitempty"`
-	Uris            []string         `json:"uris,omitempty" yaml:"uris,omitempty"`
 	Methods         []string         `json:"methods,omitempty" yaml:"methods,omitempty"`
 	EnableWebsocket bool             `json:"enable_websocket,omitempty" yaml:"enable_websocket,omitempty"`
+	RemoteAddr      string           `json:"remote_addr,omitempty" yaml:"remote_addr,omitempty"`
 	RemoteAddrs     []string         `json:"remote_addrs,omitempty" yaml:"remote_addrs,omitempty"`
-	UpstreamId      string           `json:"upstream_id,omitempty" yaml:"upstream_id,omitempty"`
+	Upstream        *Upstream         `json:"upstream,omitempty" yaml:"upstream,omitempty"`
+	UpstreamID      string           `json:"upstream_id,omitempty" yaml:"upstream_id,omitempty"`
 	ServiceID       string           `json:"service_id,omitempty" yaml:"service_id,omitempty"`
 	Plugins         Plugins          `json:"plugins,omitempty" yaml:"plugins,omitempty"`
-	PluginConfigId  string           `json:"plugin_config_id,omitempty" yaml:"plugin_config_id,omitempty"`
+	PluginConfigID  string           `json:"plugin_config_id,omitempty" yaml:"plugin_config_id,omitempty"`
 	FilterFunc      string           `json:"filter_func,omitempty" yaml:"filter_func,omitempty"`
+	Script          string           `json:"script,omitempty" yaml:"script,omitempty"`
+	ScriptID        string           `json:"script_id,omitempty" yaml:"script_id,omitempty"`
+	Status          int              `json:"status,omitempty" yaml:"status,omitempty"`
```

Comment:
It's also for API7, right?

Reply:
script and script_id are fields created by dashboard for plugin orchestration